Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -179,19 +184,102 @@ final class ChangeEmailViewModelTests: TestCase {
   }
 
   func testResendVerificationStackViewIsHidden_IfEmailIsVerified() {
```

Comment:
I would just rename the tests to testResendVerificationEmailViewIsHidden...`, since we are not using stackview anymore.